### ISSUE TITLE
add -c option to wurm

### ISSUE THREAD

**Issue by davidhbrann** [wingillis/init-scripts#2]
-n fails sometimes on interactive jobs

**Comment by davidhbrann** [comment]
Circling back to this. Should `-c` be the default? I use srun for most interactive jobs, where I typically want multiples cores on the same node rather than multiple nodes.

**Comment by wingillis** [comment]
Yeah - I think it's worth making `-c` the default, and making it an option instead of a flag, and making `-n` an option too, that's by default off

**Comment by wingillis** [comment]
reopening until I change this
